e2e/assets/metrics/fleetcontroller_service.yaml

```diff
@@ -2,10 +2,18 @@ apiVersion: v1
 kind: Service
 metadata:
   name: {{ .Name }}
+  labels:
+    app: fleet-controller
+    env: test
 spec:
   selector:
     app: fleet-controller
-    shard-default: "true"
+    {{- if .IsDefaultShard }}
+    shard-default: "{{ .IsDefaultShard }}"
+    {{- end }}
+    {{- if ne .Shard "" }}
```

Comment:
:thinking: isn't this equivalent to `{{- if not .IsDefaultShard }}`, so to an `else` block in this context?
Do we use `IsDefaultShard` to keep a non-empty default shard as a possibility?

Reply:
No, a non-empty default shard is no possibility. This should be simplified.